app/assets/javascripts/client/directives/assessments/assessmentLinks.js

```diff
@@ -75,12 +76,18 @@ PDRClient.directive('assessmentLinks', [
             };
 
             $scope.requestAccess = function() {
-              $scope.modal = $modal.open({
-                templateUrl: 'client/views/modals/request_access.html',
-                scope: $scope,
-                windowClass: 'request-access-window',
-                opened: $scope.addRequestPopover()
-              });
+              if($scope.isNetworkPartner()) {
+                $scope.performAccessRequest('facilitator').then(function() {
+                  console.log("requested")
```

Comment:
This console.log seems strange to me; it does the right behavior (with the `performAccessRequest` function), but the console output doesn't serve us any purpose.